src/application.ts

```diff
@@ -189,13 +182,18 @@ export default class Application {
             !Object.keys(resourceSchema.attributes).includes(relationshipKey)
         )
     );
-
-    Object.keys(data.relationships).forEach(relationshipName => {
-      const relationships = this.serializeRelationship((data.relationships[
-        relationshipName
-      ] as unknown) as Resource | Resource[], schemaRelationships[relationshipName].type().schema.primaryKeyName);
-      data.relationships[relationshipName] = {
-        data: relationships
+    Object.keys(data.relationships).forEach(relName => {
+      const fkName = schemaRelationships[relName].belongsTo
+        ? "id"
```

Comment:
@spersico Shouldn't this use `DEFAULT_PRIMARY_KEY`?

Reply:
Noup, in line 168 we build the resources object using "id" from the proper FK. 
This forEach ensures only that all elements in the relationships object have ONLY an **id** and **type** attributes, inside a data object.